scorecard/templates/profile/profile_detail.html

```diff
@@ -99,19 +94,26 @@ <h1 class="title">{{ geography.this.short_name }}</h1>
                     <li><i class="fa fa-li fa-user" aria-hidden="true"></i> {{ official.title }} {{ official.name }}</li>
                   </ul>
                   <div class="person-contact-details">
-                    <a href="#" class="show-contact-details"><span class="icon"><i class="fa fa-plus" aria-hidden="true"></i></span> Contact Details</a>
-                    <!--
-                    <div class="contact-details">
-                      <ul class="fa-ul">
-                        <li>
-                          <i class="fa fa-li fa-phone" aria-hidden="true"></i>  <a href="#">[phone number]</a>
-                        </li>
-                        <li>
-                          <i class="fa fa-li fa-envelope" aria-hidden="true"></i> <a href="{{ muni_contact.url }}">[email address]</a>
-                        </li>
-                      </ul>
+                    <a href="#contact-details-{{ forloop.counter }}" data-toggle="collapse" class="show-contact-details show-more"><span class="icon"><i class="fa fa-plus" aria-hidden="true"></i></span> Contact Details</a>
+                    <div class="contact-details collapse" id="contact-details-{{ forloop.counter }}">
+                      {% if official.secretary %}
+                      <div>
+                        <h4>Secretary</h4>
```

Comment:
how about a hint explaining why they're getting their secretary's details like "The best way to get hold of so and so is to contact them via their secretary"..